### PR TITLE
[zh-cn]Update endpoint-slices.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 1]
```diff
@@ -497,7 +497,7 @@ EndpointSlices 还支持围绕双栈网络和拓扑感知路由等新功能的
 ## {{% heading "whatsnext" %}}
 
 <!--
-* Read [Connecting Applications with Services](/docs/concepts/services-networking/connect-applications-service/)
+* Follow the [Connecting Applications with Services](/docs/tutorials/services/connect-applications-service/) tutorial
 -->
-* 阅读[使用 Service 连接到应用](/zh-cn/docs/concepts/services-networking/connect-applications-service/)
+* 遵循[使用 Service 连接到应用](/zh-cn/docs/tutorials/services/connect-applications-service/)教程
```
Comment (chengxiangdong) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:502]:
```suggestion
* 遵循[使用 Service 连接到应用](/zh-cn/docs/concepts/services-networking/connect-applications-service/)教程
```
新链接页面不存在，还需继续使用旧链接

**Comment by windsonsea** [comment]
Please include the changes of #37386

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 2]
```diff
@@ -23,13 +23,14 @@ description: >-
...
-Kubernetes cluster. They offer a more scalable and extensible alternative to
-Endpoints.
+<!-- 
+Kubernetes' _EndpointSlice_ API provides a way to track network endpoints
+within a Kubernetes cluster. EndpointSlices offer a more scalable and extensible
+alternative to [Endpoints](/docs/concepts/services-networking/service/#endpoints).
 -->
-**端点切片（EndpointSlices）** 提供了一种简单的方法来跟踪 Kubernetes 集群中的网络端点（network endpoints）。
-它们为 Endpoints 提供了一种可扩缩和可拓展的替代方案。
+Kubernetes 的 _EndpointSlice_ API 在 Kubernetes 集群中提供了一种方法跟踪网络端点（network endpoints）。
```
Comment (javadoors) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:31]:
```suggestion
Kubernetes 的 _EndpointSlice_ API 在 Kubernetes 集群中提供了一种跟踪网络端点（network endpoints）的方法。
```

**Comment by tengqm** [comment]
@kinzhi There are conflicts to resolved, please help check.

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 3]
```diff
@@ -500,11 +500,12 @@ EndpointSlices 还支持围绕双栈网络和拓扑感知路由等新功能的
...
-* Read [Connecting Applications with Services](/docs/concepts/services-networking/connect-applications-service/)
+* Follow the [Connecting Applications with Services](/docs/tutorials/services/connect-applications-service/) tutorial
 * Read the [API reference](/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1/) for the EndpointSlice API
 * Read the [API reference](/docs/reference/kubernetes-api/service-resources/endpoints-v1/) for the Endpoints API
 -->
-* 遵循[使用 Service 连接到应用](/zh-cn/docs/concepts/services-networking/connect-applications-service/)教程
+* 遵循[使用 Service 连接到应用](/zh-cn/docs/tutorials/services/connect-applications-service/)教程
 * 阅读 EndpointSlice API 的 [API 参考](/zh-cn/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1/)
-* 阅读 Endpoints API 的 [API 参考](/zh-cn/docs/reference/kubernetes-api/service-resources/endpoints-v1/)
+* 阅读端点 API 的 [API 参考](/zh-cn/docs/reference/kubernetes-api/service-resources/endpoints-v1/)
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:509]: Why keep "EndpointSlice" as is and translate "Endpoints" to "端點"?

**Review by Sea-n — COMMENTED** [review]
For the record, this page already synced by #38221.

Please also check line 06 (weight 45 -> 60) and some minor changes in line 18.

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md (hunk 3 above)
Comment (tengqm) [reply content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:509]: Endpoints should be kept as is.

Comment (kinzhi) [reply content/zh-cn/docs/concepts/services-networking/endpoint-slices.md]: yes i modified it

**Pull request merged**